### ISSUE TITLE
Better error for missing alphabet.txt or bytes_output_mode

### ISSUE THREAD

**Issue by JRMeyer** [coqui-ai/STT#1914]
If you don't configure the alphabet, you get an error about the dims of layer 6. You should get an error about the alphabet

```
root@e02b8a0e3e26:/code# python3 train.py \
>   --train_files stt-data/clips/train.csv \
>   --dev_files stt-data/clips/dev.csv \
>   --test_files stt-data/clips/test.csv 
WARNING:tensorflow:Deprecation warnings have been disabled. Set TF_ENABLE_DEPRECATION_WARNINGS=1 to re-enable them.
Traceback (most recent call last):
  File "train.py", line 12, in <module>
    stt_train.main()
  File "/code/training/coqui_stt_training/train.py", line 1258, in main
    train()
  File "/code/training/coqui_stt_training/train.py", line 619, in train
    gradients, loss, non_finite_files = get_tower_results(
  File "/code/training/coqui_stt_training/train.py", line 417, in get_tower_results
    avg_loss, non_finite_files = calculate_mean_edit_distance_and_loss(
  File "/code/training/coqui_stt_training/train.py", line 335, in calculate_mean_edit_distance_and_loss
    logits, _ = create_model(
  File "/code/training/coqui_stt_training/train.py", line 294, in create_model
    "layer_6", layer_5, Config.n_hidden_6, relu=False
  File "/code/training/coqui_stt_training/util/config.py", line 28, in __getattr__
    raise RuntimeError(
RuntimeError: Configuration option n_hidden_6 not found in config.
```

**Comment by jeremiahrose** [comment]
This came up in the Q&A: https://github.com/coqui-ai/STT/discussions/1935

**Comment by reuben** [comment]
This was (hopefully) fixed in #1938
